libphal/phal_dump.C

```diff
@@ -170,13 +173,14 @@ void writeSbeData(const std::filesystem::path& dumpPath, uint32_t reasonCode,
 /**
  * @brief Initialize the PDBG and SBE to start collection
  * @param[in] failingUnit Position of the proc containing the failed SBE
+ * @param[in] forOdySsey Proc target type i.e.; Odyssey or normal memory chip
  *
  * Exceptions: PDBG_INIT_FAIL for any pdbg init related failure.
  *             PDBG_TARGET_NOT_OPERATIONAL for invalid failing unit
  *             HWP_EXECUTION_FAILED if the extract rc procedure is failing
  */
 struct pdbg_target* preCollection(const uint32_t failingUnit,
-				  const std::filesystem::path& dumpPath)
+				  const std::filesystem::path& dumpPath, const bool forOdySsey)
```

Comment:
instead of forOdySeey please pass the type id

Reply:
Changed